### ISSUE TITLE
Ship (with shield) was destroyed

### ISSUE THREAD

**Issue by Scooter5131** [StarQuestMinecraft/StarQuestPublic#923]
My ship was parked right outside my town (on Erilon), and today when I logged on I found only the roof remaining. It did have a shield, and has been parked there for a while, and it happened some time in the last 3 days (I was not online Friday or Saturday).

Screenshot of what I found is attached.
![2016-05-15_09 30 23](https://cloud.githubusercontent.com/assets/13090706/15279306/7834638c-1ae8-11e6-9f50-ce2fc82ef00e.png)

FYI,
I did remove the nether slabs today, and removed the cobblestone posts to prevent creeps from coming in.

Thanks,
Scooter5131

**Comment by awangatboo** [comment]
Do you have a screen shot of the "/rg info" command showing the shield?  Did you ride on someone else's ship since the last time you parked your ship?

**Comment by Scooter5131** [comment]
I just went to the area where it was parked, and here is a screenshot of the region info.

![2016-05-16_17 20 38](https://cloud.githubusercontent.com/assets/13090706/15306924/4378f868-1b94-11e6-9945-491092d18f36.png)

This is the only ship that I own/have flown on, and I have not piloted anything else. Nobody else was on the Shield, and I am the only person in my faction at this time, so there is nobody else here.

FYI,
It was on flat land when I parked it, and it almost looked like there was a creeper explosion near where it was. Also saw that somebody was in the are as large sections of land (outside faction/unprotected) were excavated near my faction.
